ban/tests/test_commands.py

```diff
@@ -84,3 +85,16 @@ def test_export_resources():
         resource['center']['coordinates'] = list(resource['center']['coordinates'])  # noqa
         assert json.loads(lines[2]) == resource
     path.unlink()
+
+
+def test_import_ignsna(staff):
+    mun_path = Path(__file__).parent / 'data/municipalities.csv'
```

Comment:
When I use the factory, a random insee is created. But I needed a known insee to make this test. This is the reason why I make it with the municipality file.
 
An other way will be to use a mock, but it will make time to me to implement it the first time.